src/assets/styles/_variables.scss

```diff
@@ -1,8 +1,8 @@
 $logo-purple: #290b53;
 
 :root {
-  --base-font-family: Roboto, -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', 'Helvetica Neue', Arial, sans-serif;
-  --code-font-family: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
+  --font-family-sans: Roboto, -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', 'Helvetica Neue', Arial, sans-serif;
+  --font-family-mono: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
```

Comment:
Is the Roboto here just incase someone has Roboto installed on their machine, seeing as we've removed Roboto fonts here? If so should we add `Roboto Mono` to mono also?

Reply:
Huh, strange, this not intended. That’s just the old value again. I had actually used the same font stack GitHub uses (i.e. `-apple-system,BlinkMacSystemFont,"Segoe UI","Noto Sans",Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji"`). I’ve fixed that now.

Very few people should’ve Roboto installed natively on their system so there isn’t much use in including it anyway.

Reply:
> Very few people should’ve Roboto installed natively on their system

😆 Yeah thats what I thought!